command/harness/dlite/gitspaces_init.go

```diff
@@ -0,0 +1,83 @@
+package dlite
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/drone-runners/drone-runner-aws/types"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/drone-runners/drone-runner-aws/command/harness"
+	"github.com/sirupsen/logrus"
+	"github.com/wings-software/dlite/client"
+	"github.com/wings-software/dlite/httphelper"
+)
+
+type GitspacesVMInitTask struct {
+	c *dliteCommand
```

Comment:
can we give a better name than c?

Reply:
Yes, can do it. I borrowed it from the CI task